tldr/hybrid_extractor.py

```
@@ -449,19 +469,19 @@ def _safe_parse(self, parser: Any, source: bytes, file_path: Path, language: str
+            logger.error("Tree-sitter parse failed for %s (%s): %s", file_path, language, e)
+            raise ParseError(file_path, language, e) from e
 
     def _try_tree_sitter(self, extractor_method, file_path: Path, language: str) -> ModuleInfo | None:
         """Try tree-sitter extraction, return None on failure to allow Pygments fallback."""
         try:
             return extractor_method(file_path)
         except (ParseError, OSError, ValueError, RuntimeError) as e:
-            logger.warning(f"Tree-sitter extraction failed for {file_path} ({language}), falling back to Pygments: {e}")
+            logger.warning("Tree-sitter extraction failed for %s (%s), falling back to Pygments: %s", file_path, language, e)
             return None
         except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)